docs/_docs/Developers/build.md

```diff
@@ -5,45 +5,91 @@ order: 1
 toc: true
 ---
 
-## Ubuntu 18.04/18.10 and Linux Mint 19
+## Ubuntu 18.04/18.10
+*Updated by [kreyren](https://github.com/Kreyren) at 24.02.2019*
 
-* Install the dependencies:
-```
+Fork https://github.com/PhoenicisOrg/phoenicis on github before building.
```

Comment:
No need to explain forking in this tutorial.

Reply:
@plata Disagree, assuming that contributors would clone upstream which seems insane..

Reply:
Why does it seem insane? Every normal user who just wants to build the application and install/execute it won't necessarily create a fork. I wouldn't create a fork too if I only want to build and install an application.

Reply:
@madoar I though that those are instructions for a developers assuming that it's present in `docs/_docs/Developer` and that build instructions are present in `/readme.md`..

Reply:
I think it depends on how we define a developer. If we assume that every developer wants to change the code and contribute back to the main repository I agree that it makes the most sense to create a fork. 

For me:
- a developer is someone who wants to build the application himself and maybe also tweak the source code a little bit. This doesn't necessarily require a fork
- a user is someone who wants to run/execute the application. For this he doesn't need/want to build the application. Normally a user would download a prebuilt/compiled executable, which he can then execute on his local system

Reply:
meh.. seems confusing you decide what to do about it..